tests/introspection/ApiTypoTest.cs

```diff
@@ -206,6 +206,7 @@ protected ApiTypoTest ()
 			"Exr",
 			"Felica", // Japanese contactless RFID smart card system
 			"Femtowatts",
+			"Fhir",
```

Comment:
I think that means "Fast Healthcare Interoperability Resources" if we care to add a comment.